### PR TITLE
Add harbor resource

### PR DESCRIPTION
**Pull request by Dlorite**

### CODE REVIEW

File: resources/harbor/exporter-config.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+apiVersion: v1
+kind: ExporterConfig
+app: "Harbor"
+version: 1.0.0
+appVersion: 
+  - 1.10
+maintainers:
+  - name: Sysdig team
+    link: https://sysdig.com/
+description: |
```
Comment (carillan81) [inline resources/harbor/exporter-config.yaml:11]: How about a link to the blogpost here?

Comment (Dlorite) [reply resources/harbor/exporter-config.yaml:11]: Have we a blog created already?

Comment (carillan81) [reply resources/harbor/exporter-config.yaml:11]: um, maybe is not published? you can arrange the url with Golo so it doesn't change

**Comment by daviddetorres** [comment]
If there are no alerts, better not to put the file. I don't know how the frontend will behave with an alert element without data...

**Pull request merged**